src/.editorconfig

```
@@ -0,0 +1,223 @@
+csharp_style_expression_bodied_methods = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_expression_bodied_operators
+csharp_style_expression_bodied_operators = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_expression_bodied_properties
+csharp_style_expression_bodied_properties = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_inlined_variable_declaration
+csharp_style_inlined_variable_declaration = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_pattern_matching_over_as_with_null_check
+csharp_style_pattern_matching_over_as_with_null_check = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_pattern_matching_over_is_with_cast_check
+csharp_style_pattern_matching_over_is_with_cast_check = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_throw_expression
+csharp_style_throw_expression = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_var_elsewhere
+csharp_style_var_elsewhere = true:suggestion
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_var_for_built_in_types
+csharp_style_var_for_built_in_types = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_var_when_type_is_apparent
+csharp_style_var_when_type_is_apparent = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#dotnet_sort_system_directives_first
+dotnet_sort_system_directives_first = true
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#dotnet_style_coalesce_expression
+dotnet_style_coalesce_expression = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#dotnet_style_collection_initializer
+dotnet_style_collection_initializer = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#dotnet_style_explicit_tuple_names
+dotnet_style_explicit_tuple_names = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#dotnet_style_null_propagation
+dotnet_style_null_propagation = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#dotnet_style_object_initializer
+dotnet_style_object_initializer = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#dotnet_style_predefined_type_for_locals_parameters_members
+dotnet_style_predefined_type_for_locals_parameters_members = true:error
+
+# http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#dotnet_style_predefined_type_for_member_access
+dotnet_style_predefined_type_for_member_access = false:error
```

According to our current rules, this should be true
```suggestion
dotnet_style_predefined_type_for_member_access = true:error
```

---

True.